app/controllers/api/v1/posts.js

```diff
@@ -1,21 +1,74 @@
 const db = global.db;
-const {Sequelize} = require('sequelize');
+const { Sequelize } = require("sequelize");
 const Op = Sequelize.Op;
 
-module.exports =  (router) => {
-
-  router.post('/', async (req, res) => {
+module.exports = (router) => {
+  router.post("/", async (req, res) => {
     req.body.userId = req.user.id;
     const post = await db.Posts.create(req.body);
-    res.json({post: post});
+    res.json({ post: post });
+  });
+
+  router.get("/", async (req, res) => {
+    const posts = await db.Posts.findAll({
```

Comment:
Add pagination using limit/offset in findAll. Sequelize provides that option. Consider limit and offset params are coming in query params.

Reply:
I have updated this in the recent commit.

Reply:
Also, added pagination (limit/offset) for the Roles model.